evalml/tests/automl_tests/test_automl.py

```diff
@@ -1530,9 +1530,12 @@ def test_results_getter(AutoMLTestEnv, X_y_binary):
 
     assert automl.results["pipeline_results"][0]["mean_cv_score"] == 1.0
 
-    with pytest.raises(AttributeError, match="set attribute"):
+    with pytest.raises(AttributeError) as atr_error:
         automl.results = 2.0
 
+    assert "set attribute" in str(atr_error.value) or "has no setter" in str(
+        atr_error.value,
+    )
```

Comment:
So the python3.11 test errors without this change. I can't find why, but it changes the text of the AttributeError raises here to `property 'results' of 'AutoMLSearch' object has no setter` rather than before, which was `can't set attribute`.

Also codecov said that this line isn't covered, but in the full report it says it's fine.